File: app/src/main/java/com/gmail/fattazzo/formula1world/domain/F1Time.java
package com.gmail.fattazzo.formula1world.domain;

/**
 * @author fattazzo
 *         <p/>
 *         date: 06/06/17
 */
public class F1Time {

    public int millis;

    public String time;

    public F1Time() {
    }

    public F1Time(String time) {
        this.time = time;
    }

    @Override
    public String toString() {
        return "Time{" +
                "millis=" + millis +
                ", time='" + time + '\'' +
                '}';
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/domain/F1Time.kt
package com.gmail.fattazzo.formula1world.domain

/**
 * @author fattazzo
 *
 *
 * date: 06/06/17
 */
class F1Time {

    var millis: Int = 0

    var time: String? = null

    constructor() {}

    constructor(time: String?) {
        this.time = time
    }

    override fun toString(): String {
        return "Time{" +
                "millis=" + millis +
                ", time='" + time + '\'' +
                '}'
    }
}
